design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,211 @@
+type Mutation {
+    recalculateNegotiableQuotePrices(
+        quote_ids: [String]!
+    ): RecalculateNegotiableQuotePricesOutput! @doc(description:" This call refreshes item prices, taxes, discounts, cart rules in the negotiable quote. Quotes that are locked for the seller will not be updated.")
+
+    requestNegotiableQuote(
```

Comment:
The naming is confusing here. What is the purpose of this mutation, what is "Negotiable quote request"? It seems to return a new empty negotiable quote.

Reply:
Since they have mentioned the process of creating a Negotiable Quote as Request Negotiable (https://docs.magento.com/user-guide/sales/quote-request.html) quote we used the same name. Can we change this to `createNegotiableQuote` ?
Please Advise. Thanks

Reply:
Changed to `createNegotiableQuote` and removed wrapper also introduced `quote_id` in `NegotiableQuote` entity.

Reply:
> Changed to createNegotiableQuote

Latest in this PR is showing `requestNegotiableQuote` - possible you didn't push your latest changes?

Reply:
Changed now. Thanks